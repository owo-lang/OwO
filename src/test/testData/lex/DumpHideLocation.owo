Tokens in this file
should **not**
display location information.

starPlatinum = take 100 $ repeat ora
theWorld = take 100 $ repeat muda
