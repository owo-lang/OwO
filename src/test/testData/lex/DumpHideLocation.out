IdentifierToken "Tokens"
IdentifierToken "in"
IdentifierToken "this"
IdentifierToken "file"
IdentifierToken "should"
OperatorToken "**"
IdentifierToken "not"
OperatorToken "**"
IdentifierToken "display"
IdentifierToken "location"
IdentifierToken "information"
DotToken
IdentifierToken "starPlatinum"
EqualToken
IdentifierToken "take"
IntegerToken 100
OperatorToken "$"
IdentifierToken "repeat"
IdentifierToken "ora"
IdentifierToken "theWorld"
EqualToken
IdentifierToken "take"
IntegerToken 100
OperatorToken "$"
IdentifierToken "repeat"
IdentifierToken "muda"
